### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by jljohnson001**
## Type Of Change
- [ ] Update documentation
- [ ] Add new event details
- [X] Add project resources

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [X] Commit message is as per guidelines in the contributor guide

### CODE REVIEW

File: resources/frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## 🧬 Beginner Resources
+
+New to front-end development? Start here.
+
+|                                                                                                        | Description                 |
+|:-------------------------------------------------------------------------------------------------------|:----------------------------|
+| [HTML](https://www.w3schools.com/html/default.asp)                                                     | Hypertext Markup Language   |
+| [CSS](https://www.w3schools.com/css/css_intro.asp)                                                     | Cascading Style Sheets      |
+
```
Comment (irenkamalova) [inline resources/frontend/README.md:17]: Let's use html tags to build a table:
< table >, < tbody >, < tr > , < td >

File: resources/frontend/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Front-End Development
+
+🚧 This page is currently under construction. Please stay tuned for exciting updates!
+In the meantime, checkout the latest events from our community below. 🚧
+
+## 🎯 Event Resources
+- [Modern Front End Ecosystem: Overview & Trends](https://www.youtube.com/watch?v=9jV6URf0I84)
```
Comment (irenkamalova) [inline resources/frontend/README.md:7]: Let's build a table with more description: 

- event link: https://www.meetup.com/women-who-code-london/events/288908790/
- the whole description 
- a few words about author

Follow example here: https://github.com/WomenWhoCode/WWCode-London/blob/main/README.md#event-resources

You can copy the image from meetUp and add it to images to create a separate column with posters

**Review by irenkamalova — CHANGES_REQUESTED** [review]

File: resources/frontend/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## 🧬 Beginner Resources
+
+New to front-end development? Start here.
+
+|                                                                                                        | Description                 |
```
Comment (irenkamalova) [inline resources/frontend/README.md:13]: Tool

Comment (bharitwal) [reply resources/frontend/README.md:13]: +1

File: resources/frontend/README.md [hunk 4]
```diff
@@ -1,17 +1,63 @@
...
-## 🎯 Event Resources
-- [Modern Front End Ecosystem: Overview & Trends](https://www.youtube.com/watch?v=9jV6URf0I84)
+## Event Resources
 
-## 🧬 Beginner Resources
+<table style= "background-color: transparent; border-color: transparent;">
+<tbody>
 
+<tr>
+<td style="1px solid black" align="left" width="20%"><img alt="Modern Front End Ecosystem" align="left" src="resources/frontend/images/ivan-kolotilov.png"/>&nbsp;</td>
```
Comment (bharitwal) [inline resources/frontend/README.md:10]: Image is not rendering correctly. 

Please check the path.

Comment (jljohnson001) [reply resources/frontend/README.md]: The pathing here is correct, but it only seemed to be resolved when updating from absolute to relative

**Review by bharitwal — APPROVED** [review]
LGTM.

Thanks for your contribution.

**Review by irenkamalova — APPROVED** [review]

**Comment by irenkamalova** [comment]
Great job! 👏

**Comment by jljohnson001** [comment]
FYI @irenkamalova and @bharitwal, looks like you will have to merge the PR as I do not have write access.

**Comment by bharitwal** [comment]
> FYI @irenkamalova and @bharitwal, looks like you will have to merge the PR as I do not have write access.

Yes , sure. I was waiting for Irena to confirm if she is happy with the changes.

**Comment by bharitwal** [comment]
fixes #36

**Pull request merged**